test/unittests/execute_call_test.cpp

```diff
@@ -652,13 +663,13 @@ TEST(execute_call, call_initial_depth)
     const auto wasm = from_hex("0061736d01000000010401600000020b01036d6f6403666f6f0000");
 
     auto module = parse(wasm);
-    auto host_foo = [](Instance& /*instance*/, const Value*, int depth) -> ExecutionResult {
```

Comment:
Oh actually why do we drop the explicit return type?

Reply:
Sure, we just used it for clarity. But either way it is fine, the thing is typechecked in instantiate.